lib/rpmsg/rpmsg_virtio.c

```diff
@@ -571,8 +571,12 @@ static void rpmsg_virtio_rx_callback(struct virtqueue *vq)
 			status = ept->cb(ept, RPMSG_LOCATE_DATA(rp_hdr),
 					 rp_hdr->len, rp_hdr->src, ept->priv);
 
-			RPMSG_ASSERT(status >= 0,
-				     "unexpected callback status\r\n");
+			if (status < 0) {
+				metal_log(METAL_LOG_ERROR,
+					  "ept %s, cb %p, return status %d\r\n",
```

Comment:
Do we need to print ept->cb function pointer address ? only printing ept->name should be enough.

Reply:
Also, you could use "ml_err" macro in place of "metal_log(METAL_LOG_ERROR",

Reply:
After some thinking, only print `ept->name` is enough. Thanks.